samplejava/src/main/java/com/example/chattutorialjava/ChannelActivity.java

```diff
@@ -39,30 +38,29 @@ public static Intent newIntent(Context context, Channel channel) {
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_channel);
+
+        // Step 0 - inflate binding
+        ActivityChannelBinding binding = ActivityChannelBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+
         String cid = getIntent().getStringExtra(CID_KEY);
         if (cid == null) {
             throw new IllegalStateException("Specifying a channel id is required when starting ChannelActivity");
         }
 
-        // Step 0 - Get View references
-        MessageListView messageListView = findViewById(R.id.messageListView);
-        ChannelHeaderView channelHeaderView = findViewById(R.id.channelHeaderView);
-        MessageInputView messageInputView = findViewById(R.id.messageInputView);
-
         // Step 1 - Create 3 separate ViewModels for the views so it's easy to customize one of the components
         ChannelViewModelFactory factory = new ChannelViewModelFactory(cid);
```

Comment:
This ctor call is broken at the moment, but should be fine with the next release of the SDK